cmd/argocd/commands/app.go

```diff
@@ -2456,28 +2456,21 @@ func checkResourceStatus(watch watchOpts, healthStatus string, syncStatus string
 	if watch.delete {
 		return false
 	}
+
+	healthBeingChecked := watch.suspended || watch.health || watch.degraded
```

Comment:
You don't have to init the variable and can just put a condition inline. But it's a small thing.

Reply:
Hmm, I thought it would be more readable that way as it is more obvious that the code is determining if one of the health conditions is being checked. Also happy to inline it if others feel differently.

Reply:
Idk, this way works too.